app/workers/eject_cargo_worker.rb

```diff
@@ -1,11 +1,7 @@
-class EjectCargoWorker
+class EjectCargoWorker < ApplicationWorker
```

Comment:
Why does it need to derive from `ApplicationWorker` ?

Reply:
Common base classes are handy places to put common functionality.  also...DHH likes them: `ApplicationController, ApplicationRecord, ApplicationJob, etc.`